client/components/inline-support-link/docs/example.jsx

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import InlineSupportLink from 'components/inline-support-link';
+
+export default class extends React.PureComponent {
+	static displayName = 'InlineSupportLink';
+
+	render() {
+		const support = {
```

Comment:
In the readme we use `inlineSupportProps`, I think if we were to use the same here that'd be good for consistency and clarity.